.github/actions/setup-greptimedb-cluster/action.yml

```diff
@@ -31,17 +31,22 @@ runs:
   using: composite
   steps:
   - name: Install GreptimeDB operator
-    shell: bash
-    run: |
-      helm repo add greptime https://greptimeteam.github.io/helm-charts/ 
-      helm repo update
-      helm upgrade \
-        --install \
-        --create-namespace \
-        greptimedb-operator greptime/greptimedb-operator \
-        -n greptimedb-admin \
-        --wait \
-        --wait-for-jobs
+    uses: nick-fields/retry@v3
+    with: 
+      timeout_minutes: 3
```

Comment:
Should we enlarge the timeout?

Reply:
No, it's only download a small yaml file.